preprocessors/collage-detector/SBRIF.py

```diff
@@ -0,0 +1,141 @@
+import math
```

Comment:
I'm assuming this file was taken from somewhere else...there should be a reference to this and its license at the top of the file.